lib/ansible/modules/system/aix_devices.py

```diff
@@ -0,0 +1,406 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2017, 2018 Kairo Araujo <kairo@kairo.eti.br>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.1',
+    'status': ['preview'],
+    'supported_by': 'community'
+}
+
+DOCUMENTATION = '''
+---
+author:
+  - Kairo Araujo (@kairoaraujo)
+module: aix_devices
+short_description: Manages AIX devices.
+description:
+  - This module discovers, defines, removes and modifies attributes of AIX
+    devices.
+version_added: "2.7"
```

Comment:
```suggestion
version_added: '2.8'
```
Update to new release.